### PR TITLE
fix(RequestBuffer): exclude RXSNP from conflict count assertion

### PR DESCRIPTION
**Pull request by Kumonda221-CrO3**

### CODE REVIEW

**Review by linjuanZ — APPROVED** [review]

**Review by linjuanZ — COMMENTED** [review]
Please provide a detailed description of this bug before this is merged.

**Comment by Kumonda221-CrO3** [comment]
* All **```conflictMask```** signals were diminished, add ```dontTouch``` for debug-ability.
* Request ```fromB``` means that which was from RXSNP channel.
* The possible maximum confict hit with acquire tasks (hitting ```req.*``` tag and set) and replace tasks (hitting ```meta.*``` tag and set) were two simultaneously, because all other requests would never be allowed to be sent from L1 (upper TL-C components). But  snoop requests from RXSNP must not be blocked by any other outstanding requests, so RXSNP was excluded from the conflict hit count assertion.

**Comment by Kumonda221-CrO3** [comment]
* All **```conflictMask```** signals were diminished, add ```dontTouch``` for debug-ability.
* Request ```fromB``` means that which was from RXSNP channel.
* The possible maximum confict hit with acquire tasks (hitting ```req.*``` tag and set) and replace tasks (hitting ```meta.*``` tag and set) were two simultaneously, because all other requests would never be allowed to be sent from L1 (upper TL-C components). But  snoop requests from RXSNP must not be blocked by any other outstanding requests, so RXSNP was excluded from the conflict hit count assertion.

**Pull request merged**
